packages/nocms-plugin-json-resources/test/createJsonResourceProvider.spec.js

```diff
@@ -0,0 +1,110 @@
+import assert from 'assert';
+import { createJsonResourceId } from '../lib/createJsonResourceProvider.js';
+import { activate } from '../lib/index.js';
+
+const fakeFiles = new Map();
+
+fakeFiles.set('fake/path/1', '{\"_id\":\"5d775b424974756c3bc704a3\",\"index\":0,\"guid\":\"211e66ee-ec86-489d-b1de-ede8435a0058\",\"isActive\":false,\"balance\":\"$2,581.53\",\"picture\":\"http://placehold.it/32x32\",\"age\":40,\"eyeColor\":\"brown\",\"name\":\"Gertrude Hogan\",\"gender\":\"female\",\"company\":\"DANCERITY\",\"email\":\"gertrudehogan@dancerity.com\",\"phone\":\"+1 (905) 548-2162\"}');
```

Comment:
This is way more difficult to read compared to your `expected` value.

I think a simple object is way easier to read than a map.

Reply:
I personally think the Map is ok, but the stringified json wasn't very good, so I updated that and it also uses an Object